lib/endpoints/class-wp-rest-comments-controller.php

```diff
@@ -728,9 +743,20 @@ protected function prepare_status_response( $comment_approved ) {
 	protected function prepare_item_for_database( $request ) {
 		$prepared_comment = array();
 
-		if ( isset( $request['content'] ) ) {
+		/**
+		 * Allow the comment_content to be set via the 'content' or
+		 * the 'content.raw' properties of the Request object.
+		 */
+		if ( isset( $request['content']['raw'] ) ) {
+			$prepared_comment['comment_content'] = $request['content']['raw'];
+		} elseif ( isset( $request['content'] ) ) {
 			$prepared_comment['comment_content'] = $request['content'];
 		}
+		if ( current_user_can( 'unfiltered_html' ) && isset( $prepared_comment['comment_content'] ) ) {
+			$prepared_comment['comment_content'] = wp_filter_post_kses( $prepared_comment['comment_content'] );
+		} elseif ( isset( $prepared_comment['comment_content'] ) ) {
+			$prepared_comment['comment_content'] = wp_filter_kses( $prepared_comment['comment_content'] );
```

Comment:
@rmccue Turns out the problem stems from authenticating as an `admin` level user and then updating a comment for a subscriber.  I don't have easy access to a PHP 5.2 or 5.3 machine to debug this fully.  Any thoughts?